MANUAL.txt

```diff
@@ -1265,7 +1265,7 @@ header when requesting a document from a URL:
     - `-t latex` or none: `pdflatex` (other options: `xelatex`, `lualatex`,
         `tectonic`, `latexmk`)
     - `-t context`: `context`
-    - `-t html`:  `wkhtmltopdf` (other options: `prince`, `weasyprint`)
+    - `-t html`:  `wkhtmltopdf` (other options: `prince`, `weasyprint`; see [print-css.rocks](https://print-css.rocks) for a good introduction to PDF generation from HTML/CSS.)
```

Comment:
Line needs wrapping to stay within 80 cols.

Reply:
ah yes, didn't see it in the github gui... fixed.